### DISCUSSION TITLE
Delete Organization

### DISCUSSION THREAD

**Discussion by BasitAhmed** [community/community#30539] [New to GitHub]
Hi there, 
I am not more part of the "PakLimousineServices" Organization and I want to delete it from my profile. It says please contact us. Pls Help me out this. 
Thank you :)

**Comment by airtower-luna** [comment]
I don't see any organization on your profile. Maybe it's related to private activity, so it's only visible to you?

**Reply by BasitAhmed** [reply]
I am new here, I have created account years ago, then didn't use it. You can see here in attached file, yes I think it is private and only visible to me.
![organisation](https://user-images.githubusercontent.com/5801691/186181917-e7ae28a2-6648-4f30-8313-5191c3b33d0b.jpg)

**Reply by airtower-luna** [reply]
Ah, so the organization shows up in your settings, not on your profile. Does the "Leave" button for the organization work?

If not, that's something you'll have to [contact GitHub support](https://support.github.com/request) about, us other users can't help with account details.

**Reply by BasitAhmed** [reply]
No, The "Leave" button doesn't work for me. alright then let me try to contact the support then I will get back to you.

**Comment by Ra55ZZ1** [comment]
The best way will be contact github support and explain your situation , they might help you .
